### PR TITLE
biggest of n 

### PR DESCRIPTION
**Pull request by tbhaxor**
**Fixes issue:** NULL

**Changes:** Original file was empty, I wrote the code :smile:

### CODE REVIEW

File: code/unclassified/src/biggest_of_n_numbers/biggest_of_n.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Part of cosmos from opengenus foundation
+
+const biggestOfN = array => {
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/biggest_of_n.js]: Why not just use an arrow statement without braces?

```Javascript
const biggestOfN = array => Math.max.apply(null, array);
```
The return is implicit in this.

File: code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp [hunk 2]
```diff
@@ -0,0 +1,24 @@
+// Part of cosmos from opengenus foundation
+
+#include <iostream>
+#include <vector>
+#include <algorithm>
+using namespace std;
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp]: Don't use using namespace std.

File: code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+int main() {
+    int n;
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp]: I would keep entering numbers until an EOF. i.e:

```C++
std::vector<int> arr;
std::cout << "Keep entering numbers (EOF to stop): ";
for (int num; std::cin >> num;)
{
    arr.push_back(num);
}
```

File: code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    cin>>n;
+    for(int i = 0 ; i < n ; i++) {
+        int a;
+        cin>>a;
+        arr.push_back(a);
+    }
+
+    sort(arr.begin(), arr.end());
+
+    cout<<"biggest number : "<<arr[n-1];
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp]: `arr.back()` is clearer here.

File: code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+    return 0;
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp]: No need for this, it is implicit;

File: code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+}
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp]: You should probably add a newline at the EOF (By unix definition).

Comment (tbhaxor) [reply code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp]: you mean 25th line ?

Comment (arnavb) [reply code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp]: Yup

File: code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp (hunk 5 above)
Comment (tbhaxor) [reply code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp]: you mean 
```cpp
return 0;
```

File: code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp [hunk 7]
```diff
@@ -0,0 +1,17 @@
+// Part of cosmos from opengenus foundation
+
+#include <iostream>
+#include <vector>
+#include <algorithm>
+
+int main() {
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp]: Braces go on their own lines.

File: code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+    std::vector<int> arr;
+    std::cout << "Keep entering numbers (EOF to stop): ";
+    for (int num; std::cin >> num;) {
+        arr.push_back(num);
+    }
+
+    sort(arr.begin(), arr.end());
+
+    std::cout<<"biggest number : "<<arr.back();
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp]: Add spaces around operators.

File: code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+#include <algorithm>
+
+int main() {
+    std::vector<int> arr;
+    std::cout << "Keep entering numbers (EOF to stop): ";
+    for (int num; std::cin >> num;) {
+        arr.push_back(num);
+    }
+
+    sort(arr.begin(), arr.end());
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp]: Again, I suggest you actually implement the algorithm.

Comment (tbhaxor) [reply code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp]: as you also know CPP provides us STL 
for algo contributors have added the same program in C

**Comment by tbhaxor** [comment]
@AdiChat

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**